README.md

```diff
@@ -14,11 +14,45 @@ dependencies:
 **2:** Install packages from the command line:
 
 ```bash
-$ flutter packages get
```

Comment:
typically ive seen the $ being used to represent running with user permissions (as opposed to `#` when running as root). do you think its worth clarifying this another way since the $ is being removed?

Reply:
and maybe consider to change the command to `flutter pub get`, since i can't find official documentation about `flutter packages get`, and to be honest I only found out about `flutter packages get` now:)

Reply:
done